cmd/handler-utils.go

```diff
@@ -176,14 +176,26 @@ func getRedirectPostRawQuery(objInfo ObjectInfo) string {
 	return redirectValues.Encode()
 }
 
+// Returns access key in the request Authorization header.
+func getReqAccessKey(r *http.Request, region string) (accessKey string) {
+	accessKey, _, _ = getReqAccessKeyV4(r, region)
+	if accessKey == "" {
+		accessKey, _, _ = getReqAccessKeyV2(r)
+	}
+	return accessKey
+}
+
 // Extract request params to be sent with event notifiation.
 func extractReqParams(r *http.Request) map[string]string {
 	if r == nil {
 		return nil
 	}
 
+	region := globalServerConfig.GetRegion()
 	// Success.
 	return map[string]string{
+		"region":          region,
```

Comment:
The title says `Support audit logs to print accessKey and version`, but  the fix seems to also add `region`.

Before the  fix:
```
{
  "api": {
    "args": {},
    "name": "ListBuckets"
  },
  "deploymentid": "16518dd0-a4b6-4255-9e8b-b70d4a5b650b",
  "remotehost": "192.168.1.124",
  "requestID": "156314F71F07E883",
  "time": "2018-11-01T18:55:49.101220379Z",
  "userAgent": "Minio (linux; amd64) minio-go/v6.0.3 mc/2018-08-09T00:05:19Z"
}
```

After the fix:
```
{
  "api": {
    "args": {},
    "name": "ListBuckets"
  },
  "deploymentid": "16518dd0-a4b6-4255-9e8b-b70d4a5b650b",
  "metadata": {
    "accessKey": "minio",
    "region": "",
    "sourceIPAddress": "192.168.1.124"
  },
  "remotehost": "192.168.1.124",
  "requestID": "1563144747FEF874",
  "time": "2018-11-01T18:43:13.874163087Z",
  "userAgent": "Minio (linux; amd64) minio-go/v6.0.3 mc/2018-08-09T00:05:19Z",
  "version": "1"
}
```
Do we need to change the title or the fix?

Reply:
@harshavardhana,
Never  mind. You've already changed the title.